src/Raven.Server/Documents/TcpHandlers/SubscriptionDocumentsFetcher.cs

```diff
@@ -33,7 +33,7 @@ public class SubscriptionDocumentsFetcher
         private readonly SubscriptionState _subscription;
         private readonly SubscriptionPatchDocument _patch;
         // make sure that we don't use too much memory for subscription batch
-        private readonly Size _maximumAllowedMemory;
+        private readonly long _maximumAllowedMemory;
```

Comment:
Why change it to `long`?

Reply:
so you won't have to do size -> long and long and then allocate new size.
I can't accumulate the size of the buffers I only got the total and I don't want to add the total to the size each time I check the size, those are too many allocations it is better to use long.

Reply:
As commented above - `Size` is a struct. I'm for using `Size` everywhere across the board (where possible) since it makes the code less error prone.

Reply:
I think you missed changing this one